api/v1/helpers/verbs/doFind.js

```diff
@@ -71,10 +71,14 @@ function doFindResponse(reqResNext, props, opts, resultObj) {
   .then((retval) => {
     u.sortArrayObjectsByField(props, retval);
 
+    // Merge props and opt fields to exclude
+    const fieldsToExclude = [].concat(opts.fieldsToExclude || [],
+      props.fieldsToExclude || []);
```

Comment:
why do we need to use both? why not just use the existing props.fieldsToExclude?

Reply:
The method who is generating the list of attributes is utils.buildFieldList - there is supposedly the place to generate a list of attribute and now the list to remove (I have added the model as an argument. Would you change the model by reference inside that method? To me, it will do more than just build a list of Sequelize attributes (including or excluding) and might cause a side effect.

Is it ok changing object by reference as a refocus design definition?